### ISSUE TITLE
Has linuxglobalimage changed?

### ISSUE THREAD

**Issue by jzmaddock** [CPPAlliance/drone-ci#7]
See failures here: https://drone.cpp.al/boostorg/config/43/30/2  All the clang installs are currently failing.

I tried updating to image="cppalliance/droneubuntu2004:1" but that just led to further errors.

Note that the failures:

```
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 clang-10 : Depends: libclang-common-10-dev (= 1:10.0.1~++20210308124217+ef32c611aa21-1~exp1~20210308114816.190) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

Which is normally down to apt being in a pickle after an upgrade, but these should be clean images I assume?

**Comment by sdarwin** [comment]
The same problem occurs on plain docker images, not just the cppalliance versions.   Upgrading from 16.04 to 18.04 seems to help.

**Comment by jzmaddock** [comment]
I solved it by specifying the version at  18.04.  Thanks!
